index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <img src="images.svg\image_geometry_1 (1).svg" alt="" class="top-left">
```

Comment:
- [ ] Kindly make this a `background-image` in your style sheet rather than an image in the HTML markup. If an image is not part of the content (for aesthetic reasons) use a CSS `background-image.` [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. 💻 👍

Reply:
i repeated this week last year and no other code reviewer complained about this to me my page looks fine as long as it is not a semantic issue im just resubmitting code i didi since last year , again i feel youre over-scrutinizing my work  @Gambit142

Reply:
It is bad practice and is not advisable at all to have background images in the HTML file.
Always we need to design our app from the CSS file.
Chose the container that you want to style with the image, and use the `background-image` attribute in the CSS file for it.

We are allowed to use images in the HTML only if they are related to the content we display on the UI (e.g. the images of the project we build - it will be in the next milestone).

Kindly, follow the advice of the reviewer, and implement this important change.